KebPOS/Services/ProductService.cs

```diff
@@ -1,180 +1,211 @@
-                    Price = 3.99m
-                },
-                new Product()
-                {
-                    Id = 10,
-                    Name = "Lamb Kebab",
-                    Description = """
-                        Lamb Kebabs are very easy to make. 
-                        What all you need to do is marinate the mince meat with all the spices. 
-                        You can add egg also just to enhance the taste of Kebab.
-                    """,
-                    Price = 3.79m
-                },
-                new Product()
-                {
-                    Id = 11,
-                    Name = "Cappuccino",
-                    Description = "A freshly pulled shot of espresso layered with steamed whole milk and thick rich foam to offer a luxurious velvety texture and complex aroma.",
-                    Price = 1.49m
-                },
-                new Product()
-                {
-                    Id = 12,
-                    Name = "Red Bull",
-                    Description = "Red Bull is a utility drink to be taken against mental or physical weariness or exhaustion.",
-                    Price = 1.87m
-                },
-                new Product()
-                {
-                    Id = 13,
-                    Name = "Coca Cola",
-                    Description = "Coca-Cola is a carbonated, sweetened soft drink and is the world's best-selling drink. A popular nickname for Coca-Cola is Coke.",
-                    Price = 0.93m
-                },
-                new Product()
-                {
-                    Id = 14,
-                    Name = "Sprite",
-                    Description = "Crisp, refreshing and clean-tasting, Sprite is a lemon and lime-flavoured soft drink.",
-                    Price = 1.99m
-                },
-                new Product()
-                {
-                    Id = 15,
-                    Name = "Bonaqua Sparkling",
-                    Description = "BonAqua is a high-quality drinking water.",
-                    Price = 1.24m
-                }
-            };
-            return products;
+            bool duplicate = false;
+            var product = new Product();
+            product = new Product();
+            product.Name = AnsiConsole.Ask<string>("Product's name:");
+            product.Price = AnsiConsole.Ask<decimal>("Product's price:");
+            product.Description = AnsiConsole.Ask<string>("Product's description");
+            duplicate = Validation.CheckProductDuplicate(product);
+
+            if (duplicate)
+            {
+                Console.Clear();
+                Console.WriteLine($"ERROR: There is a product already named {product.Name}.  The product could not be entered.");
+                Console.ReadKey();
+            }
+            else
+            {
+                KebabController.AddProduct(product);
+            }
+
+            inOption = AnsiConsole.Ask<bool>("Enter New Product?");
```

Comment:
It took me a few iterations to find out I had to actually use a boolean. Spectre.Console has a more elegant way of dealing with this: check out this example from our Flashcard app:

![image](https://github.com/TheCSharpAcademy/CONSOLE.PointOfSale/assets/108711476/95c7f874-95d4-4d74-a2d2-1665e2cfa98e)